components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -240,23 +240,38 @@ def validate_db_config(clp_config: CLPConfig, data_dir: pathlib.Path, logs_dir:
     try:
         validate_path_could_be_dir(data_dir)
     except ValueError as ex:
-        raise ValueError(f"database data directory is invalid: {ex}")
+        raise ValueError(f"{DB_COMPONENT_NAME} data directory is invalid: {ex}")
 
     try:
         validate_path_could_be_dir(logs_dir)
     except ValueError as ex:
-        raise ValueError(f"database logs directory is invalid: {ex}")
+        raise ValueError(f"{DB_COMPONENT_NAME} logs directory is invalid: {ex}")
 
-    validate_port("database.port", clp_config.database.host, clp_config.database.port)
+    validate_port(f"{DB_COMPONENT_NAME}.port", clp_config.database.host, clp_config.database.port)
```

Comment:
So this is a little tricky: This string equates to "db.port" but in the config, the key should actually be "database.port". This is because the database component name is `db`, but in the YAML config file and in `CLPConfig`, the config object is called `database`. So we need to either change the component name to `database` or change the config key to `db`. In the v0.5-prototype branch, we decided on the [former](https://github.com/y-scope/clp/blob/v0.5-prototype/components/clp-py-utils/clp_py_utils/clp_config.py#L14).